ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -31,7 +31,7 @@
   // TODO: Determine reasonable defaults here
   public static final int DEFAULT_PIVOT_DISTANCE_FROM_HEAD = 500;
   public static final float DEFAULT_FULL_VALIDATION_RATE = .1f;
-  public static final int DEFAULT_FAST_SYNC_MINIMUM_PEERS = 5;
+  public static final int DEFAULT_FAST_SYNC_MINIMUM_PEERS = 1;
```

Comment:
Is this a dev setting?

Reply:
dang it.  We really need to expose these as command line options so I stop forgetting to change that back...